src/VisualStudio/Core/Def/Implementation/FindReferences/Contexts/WithReferencesFindUsagesContext.cs

```diff
@@ -134,6 +130,10 @@ protected override Task OnReferenceFoundWorkerAsync(SourceReferenceItem referenc
                 // First find the bucket corresponding to our definition.
                 var definitionBucket = GetOrCreateDefinitionBucket(definition);
                 var entry = await createEntryAsync(definitionBucket).ConfigureAwait(false);
+                if (entry == null)
+                {
+                    return;
+                }
```

Comment:
when does this happen?  What is the user impact.  Will they not see a reference?